components/elm/bld/namelist_files/namelist_defaults.xml

```diff
@@ -267,10 +267,12 @@ ic_tod="0" sim_year="2000" glc_nec="0" use_crop=".true." irrigate=".true." nu_co
 <!-- Initial conditions for DYAMOND1, DYAMOND2 and for seasons of ENSO neutral year 2013 -->
 <finidat hgrid="ne1024np4.pg2" mask="ICOS10" ic_ymd="20160801">lnd/clm2/initdata/20220928.I2010CRUELM.ne1024pg2_ICOS10.elm.r.2016-08-01-00000.nc</finidat>
 <finidat hgrid="ne1024np4.pg2" mask="ICOS10" ic_ymd="20200120">lnd/clm2/initdata/20221218.F2010-CICE.ne30pg2_ne1024pg2.elm.r.2020-01-20-00000.nc</finidat>
+<finidat hgrid="ne1024np4.pg2" mask="ICOS10" ic_ymd="19941001">lnd/clm2/initdata/20231226.I2010CRUELM.ne1024pg2_ICOS10.elm.r.1994-10-01-00000.nc</finidat>
```

Comment:
Could also put in a default finidat for ne256pg2 for 1994-01-01-00000.nc.

Reply:
Yeah, I could do that too, but we don't rely on these at all since the compset keeps the default start date and I elected to just set via runscript. I had hoped to actually avoid creating a new compset altogether but needed to in order to turn on MOSART.

Reply:
Okay, pushed a commit to add the default for ne256pg2. It seems to get picked up.